node/src/transition_frontier/candidate/transition_frontier_candidate_actions.rs

```diff
@@ -22,10 +20,12 @@ pub type TransitionFrontierCandidateActionWithMetaRef<'a> =
 #[derive(Serialize, Deserialize, Debug, Clone, ActionEvent)]
 #[action_event(level = debug, fields(debug(hash), debug(error)))]
 pub enum TransitionFrontierCandidateAction {
+    P2pBestTipUpdate {
+        best_tip: BlockWithHash<Arc<MinaBlockBlockStableV2>>,
+    },
     #[action_event(level = info)]
     BlockReceived {
-        hash: StateHash,
-        block: Arc<MinaBlockBlockStableV2>,
+        block: ArcBlockWithHash,
```

Comment:
@binier can this be made to print the block hash? (with some custom block printer I guess)

one nice property of the old version was that the hash was printed in the logs

Reply:
```
2025-02-26T19:36:19.399621Z  INFO openmina_core::log::action: time="1740598579338852958" kind="TransitionFrontierCandidateBlockReceived" hash=3NKQ9j3CkMhCXH4Sdsf25DMUJhi6iyqTEhpya54XBzTkxuTAHEm4
```

Reply:
Moved to logger_effects: https://github.com/openmina/openmina/blob/3305941aa2668b30d8fdd1dda27a97d6eaf0e56b/node/src/logger/logger_effects.rs#L123